### DISCUSSION TITLE
Delete account linked email

### DISCUSSION THREAD

**Discussion by PhanVu6** [community/community#135148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

How do I get my account back after removing the email link?
Before that, I had nothing linked except that email.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @PhanVu6 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by everforyou** [comment]
Hi,

You can use just username to login.
Using just the username can sometimes help streamline the recovery process, especially when contacting support.

^-^
